stylesheets/partials/overrides/_google-custom-search.scss

```diff
@@ -140,6 +140,9 @@
     }
     .gsc-above-wrapper-area-container * {
         display: block !important;
+        span,em{
```

Comment:
please use line breaks in between selectors
```css
span,
em {
...
```

also did this work?

i would have expected this too look like

```scss
.gsc-above-wrapper-area-container {
  * {
    display: block !important;
  }
  
  span,
  em {
    display: inline !important;
  }
}
```

Reply:
Surprisingly that did work but Ive changed the formatting to match what you have here. Which also works.